src/util.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Get the current totals.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getCurrentTotal( cart ) {
+	if ( undefined !== cart ) {
+		const total_price = cart.cartTotals.total_price;
+		const minor_unit = cart.cartTotals.currency_minor_unit;
+
+		return total_price / Math.pow( 10, minor_unit );
+	}
+	return 15;
```

Comment:
I think this is pretty confusing as a return value. Could the preview data not be hardcoded using the `useEditorContext` hook in your block? Mixing real and preview data is dangerous IMO.

Reply:
I was returning a default value to render the progress bar component correctly in the editor, as the editor does not have the cart object. That said, I see how this default value can be confusing, and I'm now returning the default value of the `block.json` file.